packages/tools/src/tools/annotation/planarFreehandROITool/editLoopCommon.ts

```diff
@@ -52,6 +52,11 @@ function checkForFirstCrossing(
 
     const distanceIndexPairs = [];
 
+    // prevent continue if there are not the minimum of points for this op.
+    if (prevCanvasPoints.length < 2) {
```

Comment:
Good catch, but can you move this up to the top level conditional, aroun lines 43-46? I.e. we shouldn't even enter this block if this is true.